_posts/2022-10-17-10.0.0-release.md

```diff
@@ -0,0 +1,298 @@
+* LargeBinary, LargeString and LargeList arrays have been implemented for handling arrays with 64-bit offsets. This also included fixing a bug so that binary builders are correctly resettable. ([ARROW-8226](https://issues.apache.org/jira/browse/ARROW-8226), [ARROW-17275](https://issues.apache.org/jira/browse/ARROW-17275))
+* Support for Decimal256 arrays has been implemented ([ARROW-10600](https://issues.apache.org/jira/browse/ARROW-10600))
+* Automatic Endianness Conversion for non-native endianness is now an option for IPC streams ([ARROW-17219](https://issues.apache.org/jira/browse/ARROW-17219))
+* CSV Writer now supports Timestamp, Date32 and Date64 types ([ARROW-17273](https://issues.apache.org/jira/browse/ARROW-17273))
+* CSV Writer now supports custom formatting for boolean values ([ARROW-17277](https://issues.apache.org/jira/browse/ARROW-17277))
+* The Go Arrow Library now provides a FlightSQL client and server implementation ([ARROW-17326](https://issues.apache.org/jira/browse/ARROW-17326)). An example server implementation is provided for a FlightSQL server using SQLite ([ARROW-17359](https://issues.apache.org/jira/browse/ARROW-17359))
+* CSV Reader now supports schema type inference via NewInferringReader, along with options for specifying the type of some columns and skipping columns ([ARROW-17778](https://issues.apache.org/jira/browse/ARROW-17778))
+
+### Parquet
+
+* RowGroupReader.Column(index int) no longer panics if provided an invalid column index. Instead the signature has been changed to now return (PageReader, error) similar to other methods in the codebase. ([ARROW-17274](https://issues.apache.org/jira/browse/ARROW-17274))
+* Bitpacking and other internal required implementations for ppc64le have been added for the Parquet library ([ARROW-17372](https://issues.apache.org/jira/browse/ARROW-17372))
+* A bug has been fixed that caused inconsistent row information data from a table written by Athena ([ARROW-17453](https://issues.apache.org/jira/browse/ARROW-17453))
+* Fixed a bug that caused panics when writing a Nullable List of Structs ([ARROW-17169](https://issues.apache.org/jira/browse/ARROW-17169))
+* Key Value metadata in an Arrow Schema will be propagated to the Parquet file when using pqarrow even when not using StoreSchema ([ARROW-17627](https://issues.apache.org/jira/browse/ARROW-17627))
+* A memory leak when using statistics on ByteArray columns has been fixed ([ARROW-17573](https://issues.apache.org/jira/browse/ARROW-17573))
+
+
+## Java notes
+Many important features, enhancements, and bug fixes are included in this release, as are documentation enhancements, and a large number of improvements to build processes and project infrastructure. Selected highlights can be found below.
+
+#### New Features and Enhancements
+
+- JDBC driver for Flight SQL ([12830](https://github.com/apache/arrow/pull/12830))
+- JDBC Driver for Arrow Flight SQL ([13800](https://github.com/apache/arrow/pull/13800))
```

Comment:
I think these two duplicate each other? (One was for IP clearance, then another to actually get it in-tree)

Reply:
```suggestion
- JDBC Driver for Arrow Flight SQL ([13800](https://github.com/apache/arrow/pull/13800))
```